### PR TITLE
better validation for name and description

### PR DESCRIPTION
**Pull request by shrimalmadhur**
Fixes # .

### Motivation
- Adds better validation for name and description

### CODE REVIEW

File: types/operator_metadata_test.go [hunk 1]
```diff
@@ -15,8 +15,8 @@ func TestOperatorMetadata(t *testing.T) {
 		{
 			name: "Valid metadata",
 			metadata: OperatorMetadata{
-				Name:        "test",
-				Description: "test",
+				Name:        "Ethereum Utopia",
+				Description: "Madhur's first operator is best in this world",
```
Comment (joan-el) [inline types/operator_metadata_test.go:19]: haha

**Review by joan-el — DISMISSED** [review]
lgtm

File: types/operator_metadata_test.go (hunk 1 above)
Comment (shrimalmadhur) [reply types/operator_metadata_test.go:19]: this was actually the description of the first operator I setup back in November :D

File: types/operator_metadata.go [hunk 2]
```diff
@@ -188,3 +186,23 @@ func isImageURL(urlString string) error {
...
+	if len(text) == 0 {
+		return ErrEmptyText
+	}
+
+	if len(text) > 200 {
+		return ErrTextTooLong
+	}
+
+	// Regular expression to validate text
+	textPattern := regexp.MustCompile(`^[a-zA-Z0-9@#$%^&*()'_+\- =,.;:/?|~\[\]{}]*$`)
```
Comment (antojoseph) [inline types/operator_metadata.go:200]: This looks good, `regexp.MustCompile(`^[a-zA-Z0-9 .,;:?!'"\-_/()\[\]]+$`)` suggesting a more restricted version, please consider if this looks good.

**Review by antojoseph — COMMENTED** [review]
Left a suggestion

File: types/operator_metadata.go (hunk 2 above)
Comment (shrimalmadhur) [reply types/operator_metadata.go:200]: makes sense. let's start with strict and we can loosen it if needed

**Pull request merged**